src/Pages/jieunna/Login/Login.js

```diff
@@ -1,10 +1,58 @@
 import React from "react";
 import "./Login.scss";
-import { withRouter } from "react-router-dom";
+import { withRouter } from "react-router-dom"
+import { Link } from "react-router-dom"
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idValue: "",
+      pwValue: "",
+      loginBtn: true,
+    }
+  }
+
+  // handleInput = e => {
+  //   const { value, name } = e.target;
+  //   this.setState({
+  //     [name]: value
+  //   })
+  // }
+
+  handleIdValue = (event) => {
+    this.setState({
+      idValue: event.target.value
+    })
+  }
+
+  handlePwValue = (event) => {
+    this.setState({
+      pwValue: event.target.value
+    })
+  }
+
+  goToMain = () => {
+    this.props.history.push('/Mainjieun');
+  }
+
   render() {
-    return <div></div>;
+    return (
+      <div>
+        <section className="container">
+          <article className="logo">
+            <img alt="logo" src='/images/login/logo_text.png' />
+          </article>
+          <article className="input">
+            <input className="inputId" type="text" placeholder="전화번호, 사용자 이름 또는 이메일" name="id" onChange={this.handleIdValue} />
+            <input className="inputPw" type="password" placeholder="비밀번호" name="pw" onChange={this.handlePwValue} />
+            <button className={this.state.idValue && this.state.pwValue.length >= 6 ? "activated" : "deactivated"} type="button" disabled> <Link to="/Mainjieun">로그인</Link></button>
```

Comment:
```suggestion
            <button className={this.state.idValue && this.state.pwValue.length >= 6 ? "activated" : "deactivated"} type="button" disabled>
                <Link to="/Mainjieun">로그인</Link>
            </button>
```

- 부모, 자식 요소의 파악, 그리고 가독성을 위해 항상 줄 바꿈 해주세요.
- 삼항 연산자 활용 Good!
- disabled 값은 없는건가요..?
- Link 태그 활용 Good! 이번에는 onClick 이벤트로 withRouterHOC 방법 사용해서 수정해주세요!
- this.state 객체 비구조화 할당 적용해서 코드 깔끔하게 수정해볼까요?